zeppelin-server/pom.xml

```diff
@@ -515,6 +516,18 @@
         </zeppelin.daemon.package.base>
       </properties>
     </profile>
+	<profile>
+	  <id>hadoop3</id>
+	  <properties>
+        <hadoop.version>3.0.0</hadoop.version>
+	  </properties>
+	  <dependencies>
+		<dependency>
+		  <groupId>org.apache.directory.server</groupId>
+		  <artifactId>kerberos-client</artifactId>
+		  <version>${kerberos-client.version}</version>
+		</dependency>
+	  </dependencies>
+	</profile>
```

Comment:
Why add hadoop3 profile in zeppelin-server ? It is alrready in zeppelin-zengine. And only zeppelin-zengine use hadoop.

Reply:
This Jar file require only when hadoop3 is enabled, so we have added this dependency for hadoop3 profile.

Reply:
What I mean is that zeppelin-server doesn't require hadoop3, it is zeppelin-zengine that needs hadoop3. So you should put it in zeppelin-zengine